### PR TITLE
feat: enable new homepage

### PR DESCRIPTION
**Pull request by shelegdmitriy**
This PR enables logged in gateway view as a logged out view (other words makes this view by default). At this moment, this view enabled via `NEXT_PUBLIC_SIDEBAR_LAYOUT` env variable.

### This PR will affect **only** dev.near.org

Closes https://github.com/near/near-discovery/issues/1145

### CODE REVIEW

**Comment by charleslavon** [comment]
thanks @shelegdmitriy. How should we test this PR?

**Comment by shelegdmitriy** [comment]
Good question! According to your comment [here](https://github.com/near/near-discovery/issues/1145#issuecomment-2079727420) this layout change shouldn't affect current near.org. To achieve this approach, I use  `NEXT_PUBLIC_SIDEBAR_LAYOUT ` feature flag presented by @calebjacob. To test this PR you need this:

1. **near-discovery** and **near-discovery-testnet** preview links should **not** differ from current near.org.
2. **dev-near-org** preview link should be different from current near.org and **near-discovery** and **near-discovery-testnet** preview links because I've added a `NEXT_PUBLIC_SIDEBAR_LAYOUT ` env variable for this preview link.

Sidebar itself includes a couple of minor changes:

1. Adding `<i class="ph-bold ph-arrow-square-out" />` for every external link;
2. Adding some css improvements for _Resources_ section;

**Review by charleslavon — APPROVED** [review]
:shipit:

**Comment by shelegdmitriy** [comment]
If we all good with this changes (I'd like to wait for @calebjacob approval) then here is the next steps:

1. Merge this PR, so this will be promoted into production;
2. I'll add `NEXT_PUBLIC_SIDEBAR_LAYOUT=true` for all environments on [dev-near-org](https://vercel.com/near-developer-console/dev-near-org);

Any objections/additions?

cc @charleslavon @calebjacob

**Comment by shelegdmitriy** [comment]
Hey @vanessadespain! I think you'd like to take a look at the result of this task. Checkout the `dev-near-org` preview link at the top

**Comment by vanessadespain** [comment]
Hey @shelegdmitriy is there an access that needs to be granted to view? I get an error when I click through.

**Comment by shelegdmitriy** [comment]
> Hey @shelegdmitriy is there an access that needs to be granted to view? I get an error when I click through.

Try this one https://dev-near-org-git-feat-enable-new-90a755-near-developer-console.vercel.app

**Comment by shelegdmitriy** [comment]
As per our discussion with @charleslavon, this links will be opened in a new tab:
- /blog -> https://near.org/blog;
- /events -> https://near.org/events;
- /founders -> https://near.org/founders;
- /ecosystem -> https://near.org/ecosystem;

**Review by calebjacob — APPROVED** [review]

**Pull request merged**
